service/src/main/java/uk/gov/hmcts/reform/fpl/service/SendLetterService.java

```diff
@@ -64,7 +64,10 @@ public List<SentDocument> send(DocumentReference mainDocument, List<Recipient> r
                 SendLetterResponse response = sendLetterApi.sendLetter(authTokenGenerator.generate(),
                     new LetterWithPdfsRequest(List.of(coverDocumentEncoded, mainDocumentEncoded),
                         SEND_LETTER_TYPE,
-                        Map.of("caseId", caseId, "documentName", mainDocument.getFilename())));
+                        Map.of(
+                            "caseId", caseId,
+                            "documentName", mainDocument.getFilename(),
+                            "recipients", recipients)));
```

Comment:
Think we might want just the single recipient - looks like we send an individual letter (with unique cover letter) to each recipient

Reply:
Actually I think youre right, I pushed the changes